pricelist_cache/models/product_pricelist_cache.py

```diff
@@ -192,7 +192,17 @@ def _update_pricelist_items_cache(self, pricelist_items):
     def create_full_cache(self):
         """Creates cache for all prices applied to all pricelists."""
         pricelist_ids = self.env["product.pricelist"].search([]).ids
-        # Spawn a job every 3 pricelists (reduce the number of jobs created)
+        # pricelists = self.env["product.pricelist"].search([])
+        pricelists_root = self.env["product.pricelist"]._get_root_pricelist_ids()
+        pricelist_ids = [plid for plid in pricelist_ids if plid not in pricelists_root]
```

Comment:
```suggestion
        pricelist_ids = self.env["product.pricelist"].search([("id", "not in", pricelist_root)])
```
and remove line 194, maybe?

Reply:
Updated see fixup